launch/kinesis-alerts-consumer.yml

```diff
@@ -0,0 +1,10 @@
+run:
+  type: docker
+env: []
+resources:
+  cpu: 0.1
+  max_mem: 0.1
```

Comment:
Don't forget to increase the memory to at least a gig.  This was something that bit both Ryan and myself.